client/ayon_maya/api/pipeline.py

```diff
@@ -661,6 +661,8 @@ def on_new():
     with lib.suspended_refresh():
         lib.set_context_settings()
 
+    workfile_template_builder.build_workfile_template(
+        workfile_creation_enabled=True)
```

Comment:
This will raise an error if no workfile template is enabled. We should avoid that.
```suggestion
    try:
        workfile_template_builder.build_workfile_template(
            workfile_creation_enabled=True
        )
    except TemplateProfileNotFound:
        # Note that debug messages are not enabled by default in Maya
        log.debug(
            "No workfile template profile enabled for current context. "
            "Skipping workfile creation."
        )
```

Requires import:
```python
from ayon_core.pipeline.workfile.workfile_template_builder import (
    TemplateProfileNotFound
)
```

Reply:
adressed your suggestion ✌️